stories/demos/HeaderFilters.tsx

```diff
@@ -207,15 +275,26 @@ export function HeaderFilters() {
           Clear Filters
         </button>
       </div>
-      <DataGrid
-        columns={columns}
-        rows={filteredRows}
-        enableFilterRow={enableFilterRow}
-        filters={filters}
-        onFiltersChange={setFilters}
-      />
+      <FilterContext.Provider value={filters}>
+        <DataGrid
+          columns={columns}
+          rows={filteredRows}
+          headerRowHeight={enableFilterRow ? 70 : undefined}
+        />
+      </FilterContext.Provider>
     </div>
   );
 }
 
-HeaderFilters.storyName = 'Header Filters';
+function FilterRenderer<R, SR>({
+  column,
+  children
+}: HeaderRendererProps<R, SR> & { children: (filters: Filter) => React.ReactElement }) {
+  const filters = useContext(FilterContext)!;
```

Comment:
We could make `enableFilters` as part of the filters value, then
```tsx
{filters.enabled && <div>{children(filters)}</div>}
```
🤔

Reply:
Nice 👍